### DISCUSSION TITLE
Help with animation effect

### DISCUSSION THREAD

**Discussion by stressed-pet-owner** [pmndrs/react-spring#1362] [Support]
Hello, everyone!

I want to achieve this animation, https://share.getcloudapp.com/X6uexJr6.

So far, I only got this and the "pop" effect is not really achieved: https://codesandbox.io/s/jovial-https-jg459

I also would like to know how to achieve the lines connecting to the box.

Thanks!

**Comment by tnorlund** [comment]
This is a combination of scaling the objects that 'pop' and strokeDashoffset. You can find more examples on the react-spring website.
